nav2_recoveries/src/recovery_server.cpp

```diff
@@ -45,6 +45,13 @@ RecoveryServer::RecoveryServer()
   declare_parameter(
     "plugin_types",
     rclcpp::ParameterValue(plugin_types));
+
+  declare_parameter(
```

Comment:
Please add these to our nav2_params.yaml file (and the 2 multi-robot param example files) and verify changing them works properly.